webpack.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
+  mode: 'development',
+  entry: './client/index.jsx',
+  output: {
+    path: path.resolve(__dirname, 'public'),
+    filename: 'bundle.js'
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx$/,
+        exclude: /(node_modules)/,
```

Comment:
Does node_modules still work when it's wrapped in ()?

Reply:
This just a try to make comments in PR:))
Seems it is called "regular expressions" and I need to learn what is it

Reply:
Yes, that's what's in the documentation.  Did it work for you without the parens?

Reply:
> Does node_modules still work when it's wrapped in ()?

I think it works because of these '/' symbols - reg expressions (need to investigate what is it)

Reply:
It worked for me without (). Good to know it works either way.